node/cmd/guardiand/node.go

```diff
@@ -219,7 +219,7 @@ func init() {
 	alphRPC = NodeCmd.Flags().String("alphRPC", "", "Alephium RPC URL (required)")
 	alphApiKey = NodeCmd.Flags().String("alphApiKey", "", "Alphium RPC api key")
 	alphMinConfirmations = NodeCmd.Flags().Uint8("alphMinConfirmations", 1, "The min confirmations for alephium tx")
-	alphFetchPeriod = NodeCmd.Flags().Uint8("alphFetchPeriod", 10, "The fetch events period for alwphium watcher")
+	alphPollInterval = NodeCmd.Flags().Uint8("alphPollInterval", 10, "The fetch events period for alwphium watcher")
```

Comment:
Does this mean we will fetch events every 10 seconds in production?

Reply:
No, it's just a default value, We can specify different values based on different networks.

Reply:
Let's make the default value `4 seconds`